scripts/dev/format-stackdump.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+function format() {
+sed -e "s#$GOPATH#\$GOPATH#g" -e 's/$GOPATH/\
+$GOPATH/g' -e 's/<body>/\
+<body>\
+/g' -e 's#</body>#\
+</body>#' -e 's# /usr/lib/go/src#\
+/usr/lib/go/src#g'
```

Comment:
I find something using `\n` more readable (not sure about portability):
```
sed \
-e "s#$GOPATH#\$GOPATH#g" \
-e 's/$GOPATH/\n$GOPATH/g' \
-e 's/<body>/\n<body>\n/g' \
-e 's#</body>#\n</body>#' \
-e 's# /usr/lib/go/src#\n/usr/lib/go/src#g'
```

Reply:
looks quite portable to me. will do this